docker_mcp/services/schedule.py

```
@@ -0,0 +1,322 @@
+"""
+Docker Cleanup Scheduling Service
+
+Business logic for managing scheduled Docker cleanup operations.
+"""
+
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)